orchestration/dags/jobs/import/import_qpi.py

```diff
@@ -56,9 +56,9 @@ def verify_folder():
     for s in stats:
         blob_list.append(s)
     if len(blob_list) > 0:
```

Comment:
ok pour moi. Juste un petit renommage et minirefacto de la fonction ça serait plus clair. (même si je sais que ce n'est pas toi qui vient de mettre ce code)

* `verify_folder` => `check_folder_is_not_empty`
* `blob_list = [ file for file in bucket.list_blobs(prefix=name)]`

Et enfin, ça pourrait être pas mal de créer des constantes pour les task_id qui sont liés à des branch_operator

genre
`END_TASK_ID = "end"`
`IMPORT_ANSWERS_TO_BQ_TASK_ID=import_historical_answers_to_bigquery`